src/main/java/duke/Task.java

```diff
@@ -0,0 +1,53 @@
+package duke;
+
+/**
+ * A Task class that represents a task inputted by user.
+ */
+public abstract class Task {
+
+    /**
+     * Constructor for the Task Class.
+     *
+     * @param description Description of task.
+     */
+    protected String description;
```

Comment:
Perhaps it would be better to put the javadocs for constructor just above the constructor method?